src/main/java/com/conveyal/file/FileStorageFormat.java

```diff
@@ -24,6 +22,7 @@
     // Also can be application/geo+json, see https://www.iana.org/assignments/media-types/application/geo+json
     // The extension used to be defined as .json TODO ensure that changing it to .geojson hasn't broken anything.
     GEOJSON("geojson", "application/json"),
+    JSON("json", "application/json"),
```

Comment:
Should we defer adding this value until we actually use it?

Reply:
We use it for downloading scenarios in offline mode.

Reply:
I just noticed the comment above, I will update them accordingly.

Reply:
For future reference we do actually use this. Not the enum value by name in Java code, but com.conveyal.file.FileStorageFormat#fromFilename uses file names to look up the enum values. For now it's only used to set the correct MIME type and only affects local usage. It may break with certain file types where the enum value does not match the extension field, so we'll follow up on this in another issue.